.travis.yml

```diff
@@ -11,14 +11,31 @@ before_install:
   - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then pip3 install --upgrade pip; fi
 
 env:
-  - TF_VERSION=1.1.0
-  - TF_VERSION=1.3.0
+  matrix:
+    - TF_VERSION=1.1.0
+    - TF_VERSION=1.3.0
+    - USE_OPEN_MPI=true
```

Comment:
Rename to `MPI=OpenMPI`, `MPI=MPICH` and then branch based on that?  So it's easier to add the third option later, and better visibility on Travis page.

Reply:
Did that